README.md

```diff
@@ -103,6 +103,7 @@ The output hdf5 files each contain two datasets:
 | `prompt_postfix` | str | 'None' | | text to add after the prompt, for chatML conventions use (e.g. "\<bot\>:") |
 | `disable_space_separator` | bool | False | Include flag for True, no arguments |  If you include this flag, NO spaces will be prepended to the completion. (If you do not add this flag then a space is added to every completion if it does not already have a space). Including this flag is dangerous and not recommended because if you have input data like {"prompt": "hello." "completion": "how are you?"}, when the prompt and completion are combined it will look like "hello.how are you?" which will mess up the tokenization.--completion_keyword='target'. |
 | `keep_prompt_only_sequences` | bool | False | Include flag for True, no arguments | If you include this flag, packed sequences with only prompt tokens will not be dropped. Data with only prompt will be dropped by default because training with prompt-only sequences with prompt_loss_weight=0.0 may lead to errors. Data is dropped because of one of the following conditions: 1. the input file data prompt completion pairs contains only a prompt. 2. If the sequence is truncated such that only prompt tokens remain |
+| `categories` | bool | False | If you include this flag, then the 'category' field from your input jsonls will be stored in the 'category_id' dataset in your output hdf5 files. You need to specify all the possible categories each data point could be classified as by its 'category' key. You can pass this in as a string that is a comma separated list where each element is a category, or a path to a json file that is a list where each element is the string name of a category. |
```

Comment:
Can we make this flag more informative? Like has_category_info

Reply:
I kind of prefer categories to has_category_info, because they are specifying a list of all the categories.

Reply:
Ah I see, makes sense to me